packages/portfolio-contract/test/portfolio.flows.test.ts

```diff
@@ -941,7 +941,7 @@ test.skip('rebalance handles stepFlow failure correctly', async t => {
 });
 
 test('claim rewards on Aave position', async t => {
-  const amount = AmountMath.make(USDC, 300n);
+  const amount = AmountMath.make(USDC, 2_000_000n);
```

Comment:
consider using `withAmountUtils` in this test to make it easier to work with units. this could be

```suggestion
  const amount = usdc.units(2.0);
```
Even better you could inline the `amount` since the name isn't helping